{"uuid": "d72d4fc9-de3c-49b4-81e2-41164308d487", "children": ["71525d85-7c64-4ee0-81a0-9dc55c5620b4", "c0ee8a63-84f9-42f9-b909-c91b7151c33e", "220f333a-7c51-415c-973f-82cbe6da6649", "1c2a2cb2-c865-473d-a0df-c99314fbd2b8", "5b5bdd69-01a7-46a8-b0d5-7e0217205739", "41d2dce7-1aff-49d7-8d60-aef73be58bfe", "186c7c11-d3c4-49f8-854b-6bb45fe781df", "c43a5766-1799-4480-a1da-1ff35589c4aa", "7af98ba3-94aa-4617-9356-d9b50920a372"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_catena.py\", line 16, in get_job_details\n    peviitor_jobs_data = testutils.scrape_peviitor(company_name, 'Rom√¢nia')\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 70, in scrape_peviitor\n    response_data = TestUtils._get_request(params)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 48, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/peviitor_update.py\", line 41, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.peviitor.ro/v5/get_token/\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1714147747099, "stop": 1714147748663}], "start": 1714147747099, "stop": 1714147749036}